knitr/pool-binary-trials/pool-no-pool.Rmd

```diff
@@ -238,62 +287,77 @@ vectorized form can be up to an order of magnitude or more faster in
 some cases, depending on how many repeated calculations can be
 avoided.
 
-The actual Stan program in <code>pool.stan</code> has many more
+The actual Stan program in <code>pool.stan</code> includes many
 derived quantities that will be used in the rest of this note; see
 the appendix for the full code of all of the models discussed.
 
-We start by loading the RStan package.
+We must first instantiate a CmdStanModel object then call its <code>sample</code> method.
+To do this we first call the `cmdstan_model()` function
+which instantiates a CmdStanModel object from a file containing a Stan program.
+Because these models contain a number of [include directives](https://mc-stan.org/docs/reference-manual/includes-section.html), 
```

Comment:
```suggestion
Because these models contain a number of [include directives](https://mc-stan.org/docs/reference-manual/includes.html), 
```

With `-section` it will link to the 2_27 version (at least until we fix that redirect)

Reply:
This still needs to be changed (I think you changed a different copy of the link which I hadn't caught)